bai3.js

```diff
@@ -0,0 +1,11 @@
+getPersonInfo = (obj) => {
```

Comment:
```suggestion
getPersonInfo = (person) => {
```

Please use a meaningful name for the function parameters.